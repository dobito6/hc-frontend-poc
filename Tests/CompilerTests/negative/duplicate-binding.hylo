fun f(x: Void) {
  let x = ()
  let y = ()
  let y = ()
}
