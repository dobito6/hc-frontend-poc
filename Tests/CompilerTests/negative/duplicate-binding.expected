negative/duplicate-binding.hylo:2.7-8: error: duplicate declaration
  let x = ()
      ^
negative/duplicate-binding.hylo:3.7-8: error: duplicate declaration
  let y = ()
      ^
negative/duplicate-binding.hylo:4.7-8: error: duplicate declaration
  let y = ()
      ^
